spec/casbin/core_enforcer_spec.rb

```diff
@@ -38,6 +38,8 @@
       end
 
```

Comment:
Should we add new examples? Or activate the ignored ones?

Reply:
More examples and tests in spec/casbin/enforcer_spec.rb